crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -440,12 +442,24 @@ impl EventHandler {
                     return;
                 }
 
-                // FIXME: This is arbitrary and will not work beyond
-                // specific cases (like the just-in-time channel
-                // test). This value must be computed and we must
-                // ensure that the intercepted HTLC can actually be
-                // added to the just-in-time channel
-                let channel_value = JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT;
+                // FIXME: This will set the channel capacity to twice the amount that is
+                // transferred or the `JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT` ensuring there
+                // is enough liquidity in the channel. This is arbitrary and needs
+                // to be computed, but good enough for our test cases to support any
+                // amount to be sent.
+                let channel_value = cmp::max(
+                    JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT,
+                    expected_outbound_amount_msat * 2 / 1000,
+                );
+
+                let mut user_config = coordinator_config();
+                // We are overwriting the coordinators channel handshake configuration to prevent
+                // the just-in-time-channel from being announced (private). This is required as both
+                // parties need to agree on this configuration. For other channels, like with the
+                // channel to the faucet we want this channel to be announced (public).
+                // NOTE: we want private channels with the mobile app, as this will allow us to make
```

Comment:
Are you saying that only private channels can have 0-conf?

Reply:
yes.